recipes/recipes_emscripten/r-cli/recipe.yaml

```diff
@@ -0,0 +1,54 @@
+context:
+  version: 3.6.3
+  name: r-cli
+  r_base_version: 4.4.1
+
+package:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  url:
+    - https://cran.r-project.org/src/contrib/${{ name[2:] }}_${{ version }}.tar.gz
+    - https://cloud.r-project.org/src/contrib/${{ name[2:] }}_${{ version }}.tar.gz
+  sha256: 4295085f11221c54b1dd2b1d39a675a85dfd9f900294297567e1d36f65ac4841
+
+build:
+  number: 0
+  script:
+    - $R CMD INSTALL $R_ARGS --no-byte-compile .
+    - rm $PREFIX/lib/R/library/grDevices/libs/cairo.so
+    - rm $PREFIX/lib/R/library/tcltk/libs/tcltk.so
+    # The activation script of cross-r-base copies cairo and tcltk from
+    # the build environment.
+
+requirements:
+  build:
+    - cross-r-base >= ${{ r_base_version }}
+    - ${{ compiler('c') }}
+    - ${{ compiler('cxx') }}
```

Comment:
We don't need `compiler(cxx)`, only C code in this package.

Reply:
Arg missed that !

Reply:
Addressed.